exec/cnex/util.h

```diff
@@ -23,4 +23,11 @@ typedef int BOOL;
 #define TRUE (!FALSE)
 #endif
 
+#ifndef max
+#define max(a, b) (((a) > (b)) ? (a) : (b))
+#endif
+#ifndef min
```

Comment:
Don't include a definition unless you actually need it (min).

Reply:
Removed.